tests/README.md

```diff
@@ -0,0 +1,18 @@
+Tests are run inside a Docker container which includes all the necessary Glue/Spark dependencies and simulates the environment which the Glue jobs will be run in. A Dockerfile is included in this directory. To run tests locally, launch and attach to the docker container by referencing the following example command as a template:
```

Comment:
Might want to add that AWS credentials are required and that AWS resources will be created and torn down upon running this

Reply:
Done.